testing/integration/rpc_test.go

```diff
@@ -1,37 +1,24 @@
 package integration
 
 import (
-	"github.com/kaspanet/kaspad/app/appmessage"
-	"github.com/kaspanet/kaspad/util"
-
-	rpcclient "github.com/kaspanet/kaspad/infrastructure/network/rpc/client"
+	"github.com/kaspanet/kaspad/infrastructure/network/rpcclient"
+	"time"
 )
 
-type rpcClient struct {
-	*rpcclient.Client
-	onBlockAdded func(*appmessage.BlockHeader)
-}
+const testTimeout = 1 * time.Second
```

Comment:
This variable is in the package scope, so please change it to rpcTimeout or something else more meaningful